classes/Product.php

```diff
@@ -1707,6 +1710,21 @@ public function isNew()
         return (bool) Db::getInstance()->getValue($query, false);
     }
 
+    /**
+     * Webservice getter : get if product is new.
+     *
+     * @return int
+     */
+    public function getWsIsNew()
+    {
+        $isNew = $this->isNew();
+        if (true === $isNew) {
+            return 1;
+        }
+
+        return 0;
```

Comment:
Maybe, this can be simply like this:
```suggestion
        return $this->isNew() === true
```

What do you think?

Reply:
needs to be int or string (all other variables in the API seems to be) as a boolean false will be output as empty value resulting in the same result as the original issue atleast for the products that are not new.

Reply:
You're right.
To be totally clear, It's preferable to not write a lot of lines for this kind of function, like this:

```suggestion
        return $this->isNew() === true ? 1 : 0;
```

Specially in a 8200+ lines file!

Reply:
I prefer the code to be easy to read, besides there are no other one line statements in the file and the coding standard PS-2 has a definition for if - statements. So I don't think changing it to a one line statement is appropriate in this case?

Reply:
@boherm I see the PR still has the waiting for author label, so I'm just pinging you here just in case.

Reply:
Hi @Prestaworks, I'm sorry but in that case a ternary operator is easier to read and more cleaner 😅